src/tools/annotation/FreehandRoiTool.js

```diff
@@ -691,6 +701,18 @@ export default class FreehandRoiTool extends BaseAnnotationTool {
     preventPropagation(evt);
   }
 
+  handleKeyboardCallback(evt) {
+    const { key, ctrlKey } = evt.detail;
+    if (key === 'c' && ctrlKey) {
+      this.onContourCopyHotKeyPressed();
```

Comment:
You should add a null check in case the callback was not set.

Reply:
c50035d266e2c1e89b9552777e1d9546668a5879